### PR TITLE
Updates for sentieon modules

### PR DESCRIPTION
**Pull request by ramprasadn**
## PR checklist

The module can now accept an interval file as a parameter to restrict the analysis to specific regions. This change was necessitated because the absence of all the chromosomes in the minimal reference file caused the tests to fail. With the use of an interval file users will have more flexibility in what regions they want to restrict their analysis to. Another minor change is that dnamodelapply and dnascope will now emit a tuple containing both vcf and its index.

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR> -stub`).
- [x] Ensure the test suite passes (`nextflow run . -profile test2samples,docker --outdir <OUTDIR> -stub`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by jemten** [comment]
ready to review @ramprasadn?

**Comment by ramprasadn** [comment]
Yupe!

File: workflows/raredisease.nf [hunk 1]
```diff
@@ -135,6 +135,10 @@ workflow RAREDISEASE {
         exit 1, 'Input samplesheet not specified!'
     }
 
+    if (params.variant_caller.equals("sentieon") && !params.ml_model) {
+        exit 1, 'Machine learning model not specified!'
```
Comment (jemten) [inline workflows/raredisease.nf:139]: Is the ml model a requirement of running sentieon or is it more that it's not very smart to run without it?

**Review by jemten — APPROVED** [review]
Looks good to me. One small question about the ML model.

File: workflows/raredisease.nf (hunk 1 above)
Comment (ramprasadn) [reply workflows/raredisease.nf:139]: Its more of the latter. From what I can tell from reading the documentation, using the ml model is supposed to give much better results that one can filter on, and there is no separate license to use the model. So I don't see why anyone would run without it. Having said that, I figured we can tackle that problem if an user raises that issue at some point.

**Pull request merged**